package.json

```diff
@@ -83,6 +86,9 @@
     "warning": "^3.0.0"
   },
   "peerDependencies": {
+    "@gandi/babel-preset-gettext": "*",
+    "babel-cli": "*",
```

Comment:
@yanndinendal nope because npm 2 will install it in the node_module of the lib but we need it in the root node_modules directory